### ISSUE TITLE
Transparent PrimaryColour doesn't work.

### ISSUE THREAD

**Issue by bluEmaP1E** [libass/libass#322]
Found a black rectangle in a video , then i checked the ass file.
[V4+ Styles]
Style: `Name1`,`Fontname`,10,&H00FFFFFF,&HF0000000,&H00000000,&H00000000,-1,0,0,0,100,100,0,0,1,1,1,2,0,0,1,1
Style: `Name2`,`Fontname`,100,&HFF330F0A,&H000000FF,&H0054A41A,&H00000000,0,0,0,0,200,120,0,0,1,5,2,2,10,10,10,134 
[Events]
Dialogue: 0,0:27:25.95,0:27:27.81,`Name1`,NTP,0000,0000,0000,,{\b1\pos(44,240)}Subtitle content
Dialogue: 0,0:27:25.95,0:27:27.81,`Name2`,NTP,0000,0000,0000,,{\fscx140\fscy40\pos(44,250)}■

The  subtitle maker wants to make a wireframe with a transparent PrimaryColour and a scaled ■.
The result is not transparent but black.
![2018-12-12 8 51 17](https://user-images.githubusercontent.com/11039672/49871407-261c6000-fe51-11e8-96b5-72c4fcf9a19b.png)

XySubfilter works well.
![2018-12-12 10 02 37](https://user-images.githubusercontent.com/11039672/49874611-e60dab00-fe59-11e8-9bd1-e7a9ae622889.png)

**Comment by TheOneric** [comment]
The black colour is the shadow of the quad, see #387 . As a workaround adding `{\shad0}` before the quad does give the same result in libass as your image from XySubfilter.
Tested with:
```
[V4+ Styles]
Format: Name, Fontname, Fontsize, PrimaryColour, SecondaryColour, OutlineColour, BackColour, Bold, Italic, Underline, StrikeOut, ScaleX, ScaleY, Spacing, Angle, BorderStyle, Outline, Shadow, Alignment, MarginL, MarginR, MarginV, Encoding
Style: Name1,sans,10,&H00FFFFFF,&HF0000000,&H00000000,&H00000000,-1,0,0,0,100,100,0,0,1,0.5,0.5,2,0,0,1,1
Style: Name2,sans,100,&HFF330F0A,&H000000FF,&H0054A41A,&H00000000,0,0,0,0,200,120,0,0,1,2.5,1,2,5,5,5,134

[Events]
Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text
Dialogue: 0,0:00:02.00,0:00:05.00,Name1,NTP,0,0,0,,{\b1\pos(44,240)}test\NSubtitle content
Dialogue: 0,0:00:02.00,0:00:05.00,Name2,NTP,0,0,0,,{\fscx140\fscy40\pos(44,250)\shad0}■

```

**Comment by rcombs** [comment]
Fixed in 12cf524b831289cf67ba3432264fa01a69c3bbb5.
